app/Database/Repositories/WordFilterRepositoryInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @author    MyBB Group
+ * @version   2.0.0
+ * @package   mybb/core
+ * @license   http://www.mybb.com/licenses/bsd3 BSD-3
+ */
+
+namespace MyBB\Core\Database\Repositories;
+
+//use Illuminate\Support\Collection;
+use Illuminate\Database\Eloquent\Collection;
+use MyBB\Core\Database\Models\WordFilter;
+
+interface WordFilterRepositoryInterface
+{
+    // TODO: Implement this
+	//public function find(int $id) : WordFilter;
+	
+	public function getAll() : \Illuminate\Database\Eloquent\Collection;
```

Comment:
Again, I think there's a collection interface/contract, it's better to state that we return an interface rather than a concrete type.

It's probably also best to have a `paginate()` method and paginate them in the ACP. If you have many filters, looking through them all is difficult.